common/djangoapps/student/tests/test_credit.py

```diff
@@ -4,12 +4,13 @@
 import unittest
 import datetime
 
+import ddt
 from mock import patch
 import pytz
-
 from django.conf import settings
 from django.core.urlresolvers import reverse
 from django.test.utils import override_settings
+import mock
```

Comment:
I observe that you are only using `mock.patch` which is already imported above so no nee to import mock.